publications-2024.bib

```diff
@@ -104,14 +129,33 @@ @Article{2024:dambly.samrock.ea:geophysical
   publisher = {Oxford University Press (OUP)}
 }
 
+@Article{2024:fischer.roth.ea:more-dwr,
+  author  = {Fischer, Hendrik and Roth, Julian and Wick, Thomas and Chamoin, Ludovic and Fau, Amelie},
+  title   = {{MORe DWR}: Space-time goal-oriented error control for incremental POD-based ROM for time-averaged goal functionals},
+  journal = {Journal of Computational Physics},
+  year    = 2024,
+  volume  = 504,
+  pages   = 112863,
+  month   = may,
+  issn    = {0021-9991},
+  url     = {https://www.sciencedirect.com/science/article/pii/S0021999124001128},
+  doi     = {10.1016/j.jcp.2024.112863},
+  publisher = {Elsevier BV},
+  preprint = {https://arxiv.org/abs/2304.01140}
+}
+
 @Article{2024:goldin.magomedov.ea:study,
-  author  = { Goldin, Andrei Yu. and Magomedov, Shamil M. and Faria, Luiz M. and Kasimov, Aslan R. },
-  title   = { Study of a qualitative model for combustion waves: Flames, detonations, and deflagration-to-detonation transition },
-  journal = { Computers \& Fluids },
+  author  = {Goldin, Andrei Yu. and Magomedov, Shamil M. and Faria, Luiz M. and Kasimov, Aslan R.},
+  title   = {Study of a qualitative model for combustion waves: Flames, detonations, and deflagration-to-detonation transition},
+  journal = {Computers \& Fluids},
   year    = 2024,
   volume  = 273,
   pages   = 106213,
-  doi     = {10.1016/j.compfluid.2024.106213}
+  month   = apr,
```

Comment:
This article popped up in my newsletter, but was already part of the list. I've added some more metainformation that doi2bib provided.